File: Java/SimpleArithmaticCalculator.java
/*Write a program to create a simple arithmatic calculator in java*/
import java.util.Scanner;
public class SimpleArithmaticCalculator {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        double num1,num2,result=0;
        char operator;
        System.out.println("Enter Number 1 : ");
        num1=sc.nextDouble();
        System.out.println("Enter Number 2 : ");
        num2=sc.nextDouble();
        System.out.println("Enter the operation(+,-,*,/) : ");
        operator=sc.next().charAt(0);
        switch (operator) {
            case '+':
                result = num1 + num2;
                break;
            case '-':
                result = num1 - num2;
                break;
            case '*':
                result = num1 * num2;
                break;    
            case '/':
            if(num2 != 0){
                result = num1 / num2;
            }
            else{
                System.out.println("Error:non calculable");
            }
                break;    
            default:
                System.out.println("Enter Valid operator");
                break;
            //result=sc.nextDouble();
        }
       // result=sc.nextDouble();
        System.out.println("result=" + result);
    }    
}

File: C programs/simplearithmaticcalculator.c
#include<stdio.h>
void main()
{
	double num1,num2,result;
	char operator;
	printf("Enter two number : ");
	scanf("%lf %lf",&num1,&num2);
	printf("Select the Operation(+,-,*,/):");
	scanf(" %c", &operator);
	switch (operator)
	{
		case '+':
			result = num1 + num2;
			break;
		case '-':
			result = num1 - num2;
			break;
		case '*':
			result = num1 * num2;
			break;
		case '/':
			if(num2 != 0)
			{
				result = num1/num2;
			}
			else
			{
				printf("non calculable");
			}
			//return 1;
			break;
		default:
			printf("Error:Invalid Operator");
	}
	printf("result %.2lf",result);
}
